src/util/audit.h

```diff
@@ -7,8 +7,14 @@
 #include <c-stdaux.h>
 #include <stdlib.h>
 
+enum {
+        AUDIT_TYPE_NOAUDIT,
+        AUDIT_TYPE_AVC,
+        AUDIT_TYPE_SELINUX_ERROR,
```

Comment:
This should probably use the `UTIL_AUDIT_*` prefix to adhere to the strict namespacing.